app/src/main/java/com/hover/stax/presentation/sims/SimViewModel.kt

```diff
@@ -0,0 +1,61 @@
+package com.hover.stax.presentation.sims
+
+import android.app.Application
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import android.content.IntentFilter
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import androidx.localbroadcastmanager.content.LocalBroadcastManager
+import com.hover.sdk.api.Hover
+import com.hover.stax.domain.use_case.sims.SimWithAccount
+import com.hover.stax.domain.use_case.sims.ListSimsUseCase
+import com.hover.stax.utils.Utils
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.flow.*
+import kotlinx.coroutines.launch
+import timber.log.Timber
+
+class SimViewModel(private val listSimsUseCase: ListSimsUseCase, val application: Application) : ViewModel() {
```

Comment:
Now this view model is super simple. All it does is call the use case and it gets the SIMs with all connected accounts, it doesn't need to do any logic